README.openshift.md

```diff
@@ -1,62 +1,57 @@
 ## Changing API
 
-OpenShift is split into three major repositories:
+OpenShift is split four major repositories:
```

Comment:
into four

Reply:
OpenShift Kubernetes components. It is not OpenShift as a whole.

Reply:
Done